src/views/directMessages/containers/newThread.js

```diff
@@ -831,6 +834,15 @@ class NewThread extends React.Component<Props, State> {
 
           {!existingThreadBasedOnSelectedUsers && (
             <Grow>
+              {!existingThreadWithMessages.id && (
+                <NoThreads>
+                  <NullState
+                    icon="message"
+                    heading={`Create private direct and group messages`}
```

Comment:
Sorry I'm late to this. This line is a bit over-specific I think - especially since the copy below it expands on it.. Maybe just "Send direct messages"